### PR TITLE
Rewrite data tests to support more SQL dialects

### LINKED ISSUE #17: Data test sequence invalid syntax for Oracle DB
Hello everyone,

the builtin data test sequence runs this 2 tests:

```
--passing.sql:
select 1 as id where id = 1

--failing.sql:
select 1 as id where id = 2
```

This syntax  is not valid for Oracle DB, do you have any suggestion on how to overwrite this?

Thank you for this project, made my life a lot easier for testing the adapter. See you!

### PR DESCRIPTION
**Pull request by jtcohen6**
resolves #17, [slack thread](https://getdbt.slack.com/archives/CNNPBQ24R/p1625862584036000)

### CODE REVIEW

**Review by findinpath — APPROVED** [review]

File: pytest_dbt_adapter/builtin.py [hunk 1]
```diff
@@ -219,11 +219,17 @@ def from_dict(cls, dct, overriding=None):
 
 
 TEST_PASSING_DATA_TEST = """
-select 1 as id where id = 2
+select * from (
+	select 1 as id
+) sbq
```
Comment (findinpath) [inline pytest_dbt_adapter/builtin.py:224]: why `sbq` ? what does it mean?
I'd rather go with a generic `t` .

Comment (jtcohen6) [reply pytest_dbt_adapter/builtin.py]: Short for `subquery`, could be anything. I just renamed to `my_subquery`

**Review by findinpath — APPROVED** [review]

**Comment by jtcohen6** [comment]
@kwigley If this makes sense to you, we could cut a new patch release next week

**Review by kwigley — APPROVED** [review]

**Comment by jtcohen6** [comment]
Sorry for the delayed follow-up here. I just merged this PR, let's cut a new patch release this week.

**Comment by jtcohen6** [comment]
@kwigley When you have a minute, could you cut a new patch release of `pytest-dbt-adapter` to include this change? Totally on me for losing track of this one.

**Pull request merged**